yjit/src/options.rs

```diff
@@ -111,6 +136,7 @@ pub static mut OPTIONS: Options = Options {
     code_gc: false,
     perf_map: None,
     log: None,
+    color: NON_TTY_TERMINAL_COLOR,
```

Comment:
You're going to hate me because I can be very nitpicky, but `color` doesn't fit here because it's not a command-line option.